Xenakios/TakeRenaming.cpp

```diff
@@ -161,6 +163,12 @@ void DoRenameSourceFileDialog666(COMMAND_T* ct)
 							{
 								GetSetMediaItemTakeInfo(alltakes[j],"P_SOURCE",newsrc);
 								delete thesrc;
+
+								// delete old .reapeaks file, #1140
+								char peakfn[SNM_MAX_PATH] = "";
+								GetPeakFileName(oldname.c_str(), peakfn, sizeof(peakfn));
+								if (peakfn && *peakfn != '\0')

```

Comment:
`peakfn` (pointer to char) is always non-null here (clang emits a [warning](https://travis-ci.com/cfillion/sws/jobs/196169940#L768) for this).

```suggestion
								if (*peakfn != '\0')
```

Reply:
Caught [blindly copying code](https://github.com/reaper-oss/sws/blob/6963f7563851c8dd919db426bae825843939077f/SnM/SnM_Item.cpp#L1003)!

Changed, thanks.